docs/build-system.md

```diff
@@ -413,7 +413,7 @@ Dune offers great flexibility to specify dependencies. Another interesting
 feature are globs, that allow to simplify the configuration when depending on
 multiple files. For example:
 
-```text
+```dune
 (melange.runtime_deps
```

Comment:
These fields aren't yet supported by `dune.tmLanguage.json`, should I add them?
- melange.runtime_deps
- melange.compile_flags

Reply:
Never mind, it wasn't too hard to add them in https://github.com/melange-re/melange-re.github.io/pull/188/commits/c25587a903021cab9b85b0367110e678a1a0eba9

Reply:
For some reason i don't see `runtime_deps` when it's used alone:

<img width="403" alt="image" src="https://github.com/user-attachments/assets/837017ac-bc59-4982-8b0a-2a98ed949328">

I guess it makes sense, as it's not real "dune", but an internal snippet?

Reply:
Yeah, I noticed it after I added it. It's only highlighted if it appears inside the `library` stanza, like this:

![image](https://github.com/user-attachments/assets/1082d7de-393f-447a-83a5-70ae2a39f500)

Should I just change the snippet? Or I can try adding `melange.runtime_deps` under the `stanzas` pattern instead of the `library` pattern?

Reply:
I ended up changing the snippet to put `melange.runtime_deps` inside `library` in https://github.com/melange-re/melange-re.github.io/pull/188/commits/84b4e2e98f7340d6249c6084e69769939b04ee07.

If I put it inside the `stanzas` pattern, we probably wouldn't be able to submit our improved dune syntax file to the OCaml Platform extension.